database/worker.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package database
+
+import (
+	"database/sql"
+
+	"github.com/go-vela/types/library"
+)
+
+// Worker is the database representation of a worker.
+type Worker struct {
```

Comment:
Should this also contain information about the route(s) the worker is a part of? Thinking of situations where you potentially want to do maintenance on a subset of the workers.

Reply:
That's doable. Makes sense too!